src/uct/cuda/cuda_ipc/cuda_ipc_md.c

```diff
@@ -174,6 +175,19 @@ static ucs_status_t uct_cuda_ipc_rkey_release(uct_component_t *component,
                                               uct_rkey_t rkey, void *handle)
 {
     ucs_assert(NULL == handle);
+
+    uct_cuda_ipc_key_t *key = (uct_cuda_ipc_key_t *) rkey;
+    if (key->unmap_memhandle_func != 0 && (void *)key->d_mapped != 0) {
+        ucs_trace("unmapping addr: %p", (void *)key->d_mapped);
+        ucs_status_t status = key->unmap_memhandle_func((void *)key->d_mapped);
+        if (status != UCS_OK) {
+            ucs_fatal("failed to unmap addr: %p", (void *)key->d_mapped);
+        }
+    } else {
+        ucs_trace("unmap_memhandle_func not defined, cannot unamp addr: %p",
+                  (void *)key->d_mapped);
```

Comment:
`(void *)` -> `(void*)`

Reply:
Are you sure about this? I see a space in other casts, e.g.:

https://github.com/openucx/ucx/blob/master/src/uct/cuda/cuda_ipc/cuda_ipc_iface.c#L351
https://github.com/openucx/ucx/blob/master/src/uct/cuda/cuda_ipc/cuda_ipc_md.c#L177

Reply:
but in other places:
https://github.com/openucx/ucx/blob/0220543df0fb6c7aeb012a511dc606dcf7f8e550/src/uct/ib/mlx5/ib_mlx5.c#L339

I didn't find a requirement for this in UCX Code Style. Feel free, to not fix it

Reply:
Ok, for now I'm gonna keep it as is just to be consistent as the code in `src/uct/cuda/cuda_ipc`, but indeed, in many other places I see now without space.